sdks/python/apache_beam/io/gcp/bigquery_tools.py

```diff
@@ -131,6 +131,12 @@ def get_hashable_destination(destination):
     return destination
 
 
+def no_retry_on_invalid_input(exn):
+  if isinstance(exn, ValueError):
+    return False
+  return retry.retry_on_server_errors_and_timeout_filter(exn)
```

Comment:
I wonder if this should exist under retry.py. Thoughts?

Reply:
(I mean that whole function)

Reply:
Yeah, I was thinking the same. Looking at the source of retry.py, it would fit right in (maybe with a better name?). I'm fine with moving it there.

Reply:
I was also thinking a positive logical name (e.g. `retry_on_xyz` unlike `no_retry_on_...`) rather than a negative - but I'm not sure what it could be... I can think of `retry_if_valid_input_....` or something like that.

Reply:
I changed it to `retry_if_valid_input_but_server_error_and_timeout_filter` in the latest commit. What do you think?